disorder-jack/src/Test/QuickCheck/Jack.hs

```diff
@@ -0,0 +1,28 @@
+{-# LANGUAGE NoImplicitPrelude #-}
+-- | Can be used as a drop-in replacement for "Test.QuickCheck" in some cases.
+module Test.QuickCheck.Jack (
+    Gen
+  , forAll
+  , arbitraryBoundedEnum
+  , module X
+  ) where
+
+import           Disorder.Jack as X
+
+import           Prelude (Bounded, Enum)
+
+import           Text.Show (Show)
+
+import           Test.QuickCheck as X (Arbitrary)
+
+
+type Gen a =
+  Jack a
```

Comment:
Did you want to leave off the `a` here, that can help with transformer use (it may not work in this case)